src/stylesheets/reset.less

```diff
@@ -1,28 +0,0 @@
-/* Eric Meyer's Reset CSS v2.0 - http://cssreset.com */
```

Comment:
I opted to use the bootstrap one since it's a bit more aggressive and we're now contained inside an iframe.